cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -68,13 +68,7 @@ func extractExecutionState(
 			OutputDir: outputDir,
 		}
 
-		orderedMapMigration := mgr.OrderedMapMigration{
-			Log:       log,
-			OutputDir: dir,
-		}
-
 		migrations = []ledger.Migration{
-			orderedMapMigration.Migrate,
 			storageUsedUpdateMigration.Migrate,
 			mgr.PruneMigration,
```

Comment:
If we don't have any other migrations, we could actually skip these two as well, should not have an impact but might make spork faster.